src/neo-vm/Script.cs

```diff
@@ -53,10 +53,5 @@ public Instruction GetInstruction(int ip)
             }
             return instruction;
         }
-
-        public static implicit operator byte[](Script script)
```

Comment:
Why remove this?

Reply:
I not used

Reply:
It's a `public` method and may be used in other projects.